age-calculator/README.md

```diff
@@ -0,0 +1,31 @@
+### Age Calculator
+
+Age  Calculator App written in HTML, CSS, and JavaScript.
+
+### Use of the Project:
+You can easily calculate your age in years,months and days.
```

Comment:
Try a better description. Such as "A person's age can be calculated", because "You can easily calculate your age." doesn't make sense. A person knows their own age.

Also give daily-use examples in age, as where can an age calculator be used? Offices, Movie halls, etc??